### PR TITLE
Update Github templates

### PR DESCRIPTION
**Pull request by jacobsalmela**
Update the templates to be more useful.

### CODE REVIEW

**Comment by PromoFaux** [comment]
:shipit:

**Comment by diginc** [comment]
Maybe instead of

>  +**My familiarity with the project is as follows (check one, eg [X]):** 

Try to assess if the submission is from a full fledged sysadmin or a pi-hole hobbyist who bought yesterday with questions like: ?

- **How long have you been using pi-hole:** e.g. 1 day or 9 months
- **How long have you been using linux / raspberry-pi:** e.g. 2 years

Maybe a link to the FAQ could be added too?

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,42 @@
...
+_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._
 
+**In raising this issue, I confirm the following (please check boxes, eg [X]):**
 
-##### Actual Behaviour:
+- [ ] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
+- [ ] I have verified that the issue I am reporting can be replicated, or that the [feature](https://pi-hole.net/feature-requests/) I am suggesting [isn't already acknowledged](https://github.com/pi-hole/pi-hole/wiki).
+- [ ] I have checked that the issue I'm reporting isn't already solved and no duplicates exist in [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20) and in [opened issues](https://github.com/pi-hole/pi-hole/issues)
+- [ ] I have checked the [pull requests tab](https://github.com/pi-hole/pi-hole/pulls) for existing solutions/implementations to my issue/suggestion.
+- [ ] I have prefixed my issue title with one of the following: _[BUG], [ISSUE], [FEATURE REQUEST], [QUESTION], [OTHER]_.
```
Comment (jacobsalmela) [inline .github/ISSUE_TEMPLATE.md]: Is there a way to make them show up in the markdown file?

**Comment by PromoFaux** [comment]
Looks sexy. :shipit:

**Comment by dschaper** [comment]
:+1:

**Comment by jacobsalmela** [comment]
Approved

**Pull request merged**